classes/Hook.php

```diff
@@ -221,6 +228,8 @@ public static function getIdByName($hookName, bool $withAliases = true, bool $re
     /**
      * Return hook ID from name.
      *
+     * @return string|false Hook name, or false if it doesn't exist
```

Comment:
It returns an exception if the hook does not exist, it never returns false

Reply:
ok thanks